react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+export function Accordion({ array }) {
+  const [expand, setExpand] = useState(null);
```

Comment:
`expand` sounds like a verb. State is usually a description of the "state" of the component. So `expanded` or `expandedIndex` would be better names.